viral/run_oasis.py

```
@@ -333,13 +341,31 @@ def main(
+        if (s2p_path / "full_grosmark_oasis_preprocessed.npy").exists():
+            print(f"Already processed {mouse_name} {date}, skipping")
+            continue
+        if cached_session.wheel_freeze is None:
+            print(
+                f"No wheel freeze info for {mouse_name} {date}, haven't dealt with this in oasis yet so skipping"
+            )
+            continue
+
+        try:
+            main(s2p_path, cached_session.wheel_freeze, parallel=False, plot=False)
+        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)